docs/README.md

```
@@ -21,3 +21,133 @@ npm run docs:build
+
+### Local development
+
+1. Copy `.dev.vars.example` to `.dev.vars`:
+
+   ```bash
+   cp .dev.vars.example .dev.vars
+   ```
+
+2. Edit `.dev.vars` and fill in your test key:
+
+   ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)